tests/structures/test_exception.py

```diff
@@ -45,3 +45,28 @@ def test_raise_inside_loop(self):
                 except:
                     print(x)
         """)
+
+    def test_repr(self):
+        def escape(s):
```

Comment:
What about just putting all the test input here literally? There isn't much savings having it generated and it has the overhead for the next person to look at it of having to understand the test.

```
        self.assertCodeExecution("""
            print('>>> print(repr(ValueError(\'\')))')
            print(repr(ValueError('')))
            print('>>> print(repr(ValueError(\'"\')))')
            print(repr(ValueError('"')))
            print('>>> print(repr(ValueError("\'")))')
            print(repr(ValueError("'")))
            print('>>> print(repr(ValueError("\\"\'")))')
            print(repr(ValueError("\"'")))
            print('>>> print(repr(ValueError([1, 2, 3])))')
            print(repr(ValueError([1, 2, 3])))
            print('>>> print(repr(ValueError([1], 1, \'1\')))')
            print(repr(ValueError([1], 1, '1')))
        """")
```

Reply:
Might want to add the empty case as well here.